common/src/com/thoughtworks/go/domain/Pipeline.java

```diff
@@ -59,7 +59,7 @@ public Pipeline(String pipelineName, BuildCause buildCause, Stage... stages) {
     }
 
     public File defaultWorkingFolder() {
-        return new File(CruiseConfig.WORKING_BASE_DIR + pipelineName);
+        return new File("" + pipelineName);
```

Comment:
Makes sense not to have it (we should never have null pipeline name). The reason for it was probably only because [some tests](https://build.go.cd/go/tab/build/detail/build-linux-PR/813/build-server/1/server-split-runInstance-6) failed. My guess is that the tests are wrong. So, I'll fix them instead.